omod/src/main/java/org/openmrs/module/datafilter/web/UserFormSubmissionHandler.java

```diff
@@ -41,35 +41,35 @@
 
 @Component("userFormSubmissionHandler")
 public class UserFormSubmissionHandler {
-	
+
 	private static final Logger log = LoggerFactory.getLogger(UserFormSubmissionHandler.class);
-	
+
 	@Transactional
 	public void handle(ServletRequest request, ServletResponse response, FilterChain chain)
-	    throws IOException, ServletException {
-		
+			throws IOException, ServletException {
+
 		//Process form data from legacy UI
 		chain.doFilter(request, response);
-		
+
 		if (((HttpServletRequest) request).getMethod().equals("POST")
-		        && ((HttpServletResponse) response).getStatus() == 200) {
-			
+				&& ((HttpServletResponse) response).getStatus() < 400) {
```

Comment:
Why change to <400?

Reply:
In legacy UI we have only handled setting status on error and by default we dont get 200 we get 302 redirect if user is created successfully